qcodes/instrument_drivers/yokogawa/GS200.py

```diff
@@ -187,15 +182,16 @@ def __init__(self, name: str, address: str, terminator: str="\n",
                            set_cmd=self._set_source_mode,
                            vals=Enum('VOLT', 'CURR'))
 
-        # When getting the mode internally in the driver, look up the mode as recorded by the _cached_mode property,
-        # instead of calling source_mode(). This will prevent frequent VISA calls to the instrument. Calling
-        # _set_source_mode will change the chased value.
+        # When getting the mode internally in the driver, look up the mode as
+        # recorded by the _cached_mode property, instead of calling source
+        # _mode(). This will prevent frequent VISA calls to the instrument.
+        # Calling _set_source_mode will change the chased value.
         self._cached_mode = "VOLT"
 
-        # We want to cache the range value so communication with the instrument only happens when the set the
-        # range. Getting the range always returns the cached value. This value is adjusted when calling
-        # self._set_range
-        self._cached_range_value = None # type: Optional[Union[float,int]]
+        # We want to cache the range value so communication with the instrument
+        # only happens when the set the range. Getting the range always returns
+        # the cached value. This value is adjusted when calling self._set_range.
+        self._cached_range_value = None  # type: Optional[Union[float,int]]
```

Comment:
This type annotation can be moved to the left of the equality sign and `Optional[float]` should be equivalent